skrf/io/touchstone.py

```diff
@@ -48,6 +48,7 @@ def remove_prefix(text: str, prefix: str) -> str:
         return text[len(prefix) :]
     return text
 
+FREQ_UNITS = {"hz": 1.0, "khz": 1e3, "mhz": 1e6, "ghz": 1e9}
```

Comment:
you can add THz as it used by some users. (we have margin for PHz, EHz and ZHz...)